ADAL/tests/XCTestCase+TestHelperMethods.m

```diff
@@ -501,5 +496,136 @@ - (ADTestURLResponse *)adResponseAuthCode:(NSString *)authCode
     return response;
 }
 
+- (MSIDTokenCacheItem *)adCreateAccessMSIDTokenCacheItem
+{
+    MSIDTokenCacheItem *tokenCacheItem = [MSIDTokenCacheItem new];
+    tokenCacheItem.accessToken = TEST_ACCESS_TOKEN_TYPE;
```

Comment:
nit: can we use test identifiers from MSID?

Reply:
done